privilege-request-protocol.md

```diff
@@ -0,0 +1,208 @@
+HTTP Privilege Request Protocol
+===============================
+
+Introduction
+------------
+This memo specifies a protocol between an HTTP client and server for the
+client to request additional access privilege when denied access to a resource
+by the server for insufficient privilege.
+
+The client will typically use some form of authentication when communicating
+with the server. The method and semantics of authentication are beyond the
+scope of this memo. Methods could include, but are not limited to, use of the
+HTTP `Authorization` header using a scheme supported by the server such as
+`Bearer`, or a TLS client certificate.
```

Comment:
The way Solid would use this would at the moment strictly be with `Authorization Bearer` + PoP token, so it might be confusing to mention TLS client certificate as an example here, even if it's just an example. Then again, at the level of this document, the point stands that that's an independent choice.

Reply:
the intent of this proposal is that it's generic, and not only not solid-specific but not even specific to today's solid auth mechanisms.